ios-trivia-game/ViewControllers/HomeViewController.swift

```diff
@@ -70,7 +70,7 @@ class HomeViewController: UIViewController {
         FirebaseClient.instance.getGameRooms(complete: { (dictionary) in
             for (roomId, roomInfo) in dictionary {
                 let gameRoom = GameRoom(dictionary: roomInfo as! NSDictionary)
-                if gameRoom.state != GameRoom.State.end {
+                if gameRoom.state == GameRoom.State.idle {
```

Comment:
@plushysmurf3 As talked about, I'm checking games in idle state here to make sure the home view controller only shows games that a user can join.